doc/newfeat.texi

```diff
@@ -1,6 +1,39 @@
 @comment @itemize @w{}
 @comment @item
 
+Changes and New Features in 15.09:
+@itemize @bullet
+@item @ESS{[R]}: The indentation logic has been refactored. It should
+be faster, more consistent and more flexible. A test framework has been
+set up in order to prevent any regression on future code change.
+@item @ESS{[R]}: The most visible user change in the indentation setup
+is that customisation is now centered around 3 settings:
+@code{ess-offset-arguments}, @code{ess-offset-arguments-newline} and
+@code{ess-offset-statement}. The first two settings control how
+arguments are indented inside function calls and index brackets, while
+the latter controls the indentation of statement blocks. They all take
+either a number @code{N} (indentation relative to innermost function
+call), a list @code{'(N)} containing a number (indentation relative to
+last line indentation as in RStudio) or a @code{nil} value (indentation
+relative to opening delimiter). See documentation for more
+information. Indentation of delimiters refer to these three values
+depending on the context in order to offer a consistent indentation.
+@item @ESS{[R]}: @code{ess-arg-function-offset} has been replaced by
+@code{ess-align-from-outer-parameter}. This setting works with both
```

Comment:
I thought that `align` was more clear but you're right that naming consistency is more important